modules/about-rhde.adoc

```diff
@@ -6,45 +6,58 @@
 [id="about-rhde_{context}"]
 = About {product-title}
 
-{product-title} is for organizations who need small-form-factor edge devices with support for bare metal, virtualized, or containerized applications. {product-title} helps address many of the emerging questions around large-scale edge computing at the device edge by incorporating Kubernetes features in a new, smaller, lighter-weight footprint with an edge-optimized Linux OS built from the world's leading enterprise Linux platform in {op-system}.
+{product-title} is for organizations who need small-form-factor edge devices and support for bare metal, virtualized, or containerized applications. {product-title} helps address many of the emerging questions around large-scale edge computing at the device edge by incorporating Kubernetes features in a new, smaller, lighter-weight footprint with an edge-optimized Linux OS built from the world's leading enterprise Linux platform in {op-system}.
 
 .{product-title} components
 image::RHDevice_Edge_Overview_1.png[]
 
 .{product-title} layers
 image::RHDevice_Edge_Overview_2.png[]
 
+For a non-technical description of {product-title}, see link:https://www.redhat.com/en/technologies/device-edge[Red Hat Device Edge].
+
 [id="device-edge-relnotes_{context}"]
 == {product-title} product release notes
 
-The release notes for each product that is part of {product-title} are available at the following links:
+The latest release notes for each product that is part of {product-title} are available at the following links:
 
-* link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.13/html/release_notes/index[{microshift-first}]
-* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9-beta/html/9.2_release_notes/index[{op-system-ostree-first}] release notes contain details about {op-system-ostree}
+* link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.14/html/release_notes/index[{microshift-first}]
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.2_release_notes/index[{op-system-ostree-first}] release notes contain details about {op-system-ostree}
 
 [id="device-edge-compatibility"]
 == {product-title} product release compatibility matrix
 
 The two products of {product-title} work together as a single solution for device-edge computing. To successfully pair your products, use the verified releases together for each as listed in the following table:
 
-[cols="3",%autowidth]
+[cols="4",%autowidth]
 |===
 |*{op-system-ostree} version*
 |*{microshift} version*
 |*{microshift} release status*
+|*{microshift} supported updates*
 
-|8.7
-|4.12
-|Developer Preview
+|9.2
+|4.14
+|Generally Available
+|4.14.0-4.14.z and 4.14-4.15
```

Comment:
can add and "arrow" to make clear FROM -> TO we do support? i.e. 4.14.0 -> 4.14.z, 4.14.z -> 4.15.z

Reply:
otherwise, looks good to me!